pyro/infer/mcmc/adaptation.py

```diff
@@ -8,9 +8,11 @@
 
 import pyro
 from pyro.ops.dual_averaging import DualAveraging
-from pyro.ops.welford import WelfordCovariance
+from pyro.ops.welford import WelfordCovariance, WelfordArrowheadCovariance
 
 adapt_window = namedtuple("adapt_window", ["start", "end"])
+ArrowheadMatrix = namedtuple("ArrowheadMatrix", ["top", "bottom_diag"])
```

Comment:
Do you assume everywhere that an `ArrowheadMatrix` is symmetric? (i would expect so)